README.md

```diff
@@ -92,8 +92,8 @@ page on the EFIDroid wiki for an exact mapping of LK targets to SoCs.
 - Wileyfox Swift - crackling
 - Xiaomi Mi 4i - ferrari
 - Xiaomi Redmi 2 - wt86047, wt88047
+- Xiaomi Redmi 3 - ido
 - Yamada EveryPad III (rebrand of Lenovo PHAB Plus)
-
```

Comment:
Why did you remove this whitespace? Some markdown viewers don't render paragraphs if there isn't a blank line before them.

Reply:
It's better now?

Reply:
Yeah, looks fine now :+1: